assignments/#10.py

```diff
@@ -0,0 +1,15 @@
+num_map = [(1000, 'M'), (900, 'CM'), (500, 'D'), (400, 'CD'), (100, 'C'), (90, 'XC'),
+           (50, 'L'), (40, 'XL'), (10, 'X'), (9, 'IX'), (5, 'V'), (4, 'IV'), (1, 'I')]
+
+
+def num2roman(num):
+
+    roman = ''
+
+    while num > 0:
+        for i, r in num_map:

```

Comment:
Better to use dictionary datastucture instead of tuples list (in case if you're familiar with it)